qurro/_rank_utils.py

```diff
@@ -89,13 +89,15 @@ def differentials_to_df(differentials_loc):
 
 
 def filter_unextreme_features(
```

Comment:
Does this filter `unextreme`  features by a certain axis (if using DEICODE) or covariate (using songbird) ?

Reply:
It uses all of the feature rankings (either loading axes or differential columns). Say you specify `-x 10`, and you have three feature ranking columns (this would be the case in a normal DEICODE biplot). In this case, there would be a maximum of `60` features left in the Qurro visualization: 10 features from each end of each ranking column.

Of course, this number isn't exact: if some features overlap, they obviously won't be included twice. (We use Python `set`s to handle this sort of case automatically.) Theoretically, you could have the same extreme features for every feature ranking (so for our prior example, you could have as few as `20` features left in the visualization—that's assuming you have at least 20 features in the dataset in the first place).

So there's going to be some inherent variation in how many features are left in the visualization. The idea of imposing a stricter limit on this [has come up before](https://github.com/biocore/qurro/issues/115), but I'm not sure it's high priority enough to focus on.